src/main/webapp/script.js

```diff
@@ -97,11 +97,30 @@ function fetchAuthIndexQuestion() {
       createAuthenticationButton(
           user.authenticationUrl, 'btn-outline-success', 'Log Out', 'login');
 
-      // Show question submission box when logged in.
+      // Show submission forms when logged in.
       const questionSubmission = document.getElementById('post-question');
       if (questionSubmission) {
         questionSubmission.style.display = 'block';
       }
+
+      const answerSubmission = document.getElementById('post-answer');
+      if (answerSubmission) {
+        answerSubmission.style.display = "block";
+      }
+
+      const commentSubmission = document.getElementsByClassName('post-comment');
+      if (commentSubmission != null) {
+        // The timeout is to wait for the dynamically generated forms of each
+        // answer to appear in the DOM so that the attribute can be changed.
+        setTimeout(() => {
+          for (element of commentSubmission) {
+            element.style.display = "block";
+          }
+          // The timeout of 500ms is enough to let the forms load and not make
```

Comment:
Fine for now, but just FYI in practice you might see this fail for users with poor connections.

Reply:
yeah, I know it isn't the best practice to use timeouts and I should do some async functions but I didn't think of a flow that was appropiate